bots/inspect-queue

```diff
@@ -21,38 +21,57 @@ MAX_PRIORITY = 9
 
 import argparse
 import sys
-import time
+import ssl
 
-import amqp
+import pika
+
+seen_messages = 0
 
 def main():
     parser = argparse.ArgumentParser(description='Read and print messages from the queue without acknowleding them')
     parser.add_argument('--amqp', default='localhost',
                         help='The URL of the AMQP server to consume from')
     parser.add_argument('--queue', default='tasks',
                         help='The name of the queue to consume from')
-    parser.add_argument('-n', '--number', default=0,
-                        help='Number of queue items to show, starting from the front (0=all)')
-    parser.add_argument('-t', '--timeout', default=0.5,
-                        help='Time to wait for messages')
     opts = parser.parse_args()
 
-    connection = amqp.Connection(host=opts.amqp)
-    connection.connect()
+    host, port = opts.amqp.split(':')
+    connection = pika.BlockingConnection(pika.ConnectionParameters(
+        host=host,
+        port=int(port),
+        ssl=True,
+        ssl_options=pika.SSLOptions(
+            ssl_version=ssl.PROTOCOL_TLSv1_2,
+            cafile='/run/secrets/webhook/ca.pem',
+            keyfile='/run/secrets/webhook/amqp-client.key',
+            certfile='/run/secrets/webhook/amqp-client.pem',
+            server_hostname='amqp-cockpit.apps.ci.centos.org'),
```

Comment:
Shouldn't this be `host` again? This presumably is for SNI, but hardcoding the value here looks wrong. I don't think we need to support cases where we need SNI for a host name that isn't valid in DNS.